src/v/kafka/client/test/fixture.h

```diff
@@ -19,6 +19,25 @@ namespace kc = kafka::client;
 
 class kafka_client_fixture : public redpanda_thread_fixture {
 public:
+    kafka_client_fixture()
+      : redpanda_thread_fixture(
+        model::node_id(1),
+        9092,
+        33145,
+        8082,
+        8081,
+        43189,
+        {},
+        ssx::sformat("test.dir_{}", time(0)),
+        std::nullopt,
+        true,
+        std::nullopt,
+        std::nullopt,
+        std::nullopt,
+        configure_node_id::yes,
+        empty_seed_starts_cluster::yes,
+        std::make_optional<uint32_t>(1)) {}
```

Comment:
This change is quite subtle, and quite broad. Is there a way to limit the scope to just the retry code?

Or is the goal to broadly check that retries work?

Reply:
The goal was to add `kafka_admin_topic_api_rate` so that `throttling_quota_exceeded` error is triggered in one unit test. I'll find a way to refactor so the change is smaller.